include/dlpack/dlpack.h

```diff
@@ -38,6 +38,23 @@
 #ifdef __cplusplus
 extern "C" {
 #endif
+
+/*!
+ * \brief The DLPack version.
+ *
+ * A change in major version indicates that we have changed the
+ * data layout of the ABI - DLManagedTensorVersioned.
+ *
+ * A change in minor version indicates that we have added new
+ * code, such as a new device type, but the ABI is kept the same.
```

Comment:
I would add something of the following sort just to spell it out clearly somewhere:

> If an obtained DLPack tensor has a major version that disagrees with the version number specified in this header file (i.e. <tt>major != DLPACK_MAJOR_VERSION</tt>), the consumer must call the deleter. It is not safe to access any other fields as the memory layout will have changed. In the case of a minor version mismatch, the tensor can be safely used as long as the consumer knows how to interpret all fields. Minor version updates indicate the addition of enumeration values.

Reply:
Thank you for the great suggestion, just add the comment per suggestion

Reply:
Great suggestion +1